test/js/data/symbol_bucket.test.js

```diff
@@ -72,5 +73,19 @@ test('SymbolBucket', function(t) {
     var b2 = collision.grid.keys.length;
     t.equal(a2, b2, 'detects collision and does not place feature');
 
+    // Use a custom console.warn to count warnings
```

Comment:
Why did you put this within the existing test case rather than making a new test case?